src/components/HomePage/HothDescription.js

```diff
@@ -41,8 +43,7 @@ const useStyles = makeStyles(theme => ({
 	catchphrase: {
 		textAlign: 'center',
 		fontSize: '2.5em',
-		fontWeight: theme.typography.fontWeightBold,
-		color: theme.palette.primary.dark,
+		color: '#FF5F96',
```

Comment:
let's use `theme.palette.secondary.light` here instead so we don't have a hardcoded hex value

Reply:
Sounds good!